tools/dockerfile/distribtest/python_centos6_x64/Dockerfile

```diff
@@ -12,22 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Note: You'll need to enable vsyscall emulation in your kernel to run this image
+# locally.
 FROM centos:6
 
 RUN yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm
 
-# Vanilla CentOS6 only has python 2.6 and we don't support that.
+# This repo only supports up to Python 3.4.
```

Comment:
So... are we going to test 3.4 on CentOS6?

Reply:
It looks like the proposition is either test 3.4 or don't test CentOS 6.

Reply:
Resolved offline. Dropping the Centos 6 test.